src/pudl/helpers.py

```diff
@@ -1593,9 +1592,6 @@ def post_process(self, value: str) -> str:
         if value is None:
             try:
                 value = os.environ.get(self.env_var)
-
```

Comment:
This shouldn't be necessary since `get_settings` will set env vars if they aren't already